app/views/declaration/summary/additional_documents.scala.html

```diff
@@ -72,6 +72,7 @@
 
 @tableCaption = @{
     if(licenceRequired && showWaiverInfo) messages("declaration.summary.items.item.licences")
+    else if(declarationType == models.DeclarationType.CLEARANCE || !showWaiverInfo) messages("declaration.summary.items.item.clearance.additionalDocuments")
```

Comment:
This condition is a little confusing. If declarationType is CLEARANCE or Not(declarationType is not CLEARANCE or is999LEnabled. 
Could this be simplified to `if(declarationType == models.DeclarationType.CLEARANCE || !waiver999LConfig.is999LEnabled)` ?